funnel/assets/js/comments.js

```diff
@@ -299,13 +310,17 @@ const Comments = {
             (entries) => {
               entries.forEach((entry) => {
                 if (entry.isIntersecting) {
-                  $.ajax({
-                    url: lastSeenUrl,
-                    type: 'POST',
-                    data: {
-                      csrf_token: $('meta[name="csrf-token"]').attr('content'),
+                  const csrfToken = $('meta[name="csrf-token"]').attr(
+                    'content'
+                  );
+                  fetch(lastSeenUrl, {
+                    method: 'POST',
+                    headers: {
+                      Accept: 'application/json',
+                      'Content-Type': 'application/x-www-form-urlencoded',
+                      'X-Requested-With': 'XMLHttpRequest',
                     },
-                    dataType: 'json',
+                    body: `csrf_token=${encodeURIComponent(csrfToken)}`,
```

Comment:
Would it help if we start accepting JSON submissions in all these endpoints? I've been working on a change in Coaster. We need one more in WTForms, which can be done in Baseframe.